matter/wifi/rs911x/hal/efx_spi_intf.c

```diff
@@ -298,7 +298,7 @@ static void tx_do_dma(uint8_t *rx_buf, uint8_t *tx_buf, uint16_t xlen)
      * TODO - the caller specified 8/32 bit - we should use this
      * instead of dmadrvDataSize1 always
      */
-  if (rx_buf == UINT8_BUFF_0) {
+  if (rx_buf == (uint8_t *)NULL) {
```

Comment:
```suggestion
  if (rx_buf == NULL) {
```

Reply:
done

Reply:
you wrote done and resolve but the code didn't change